pkg/model/iam/iam_builder.go

```diff
@@ -905,6 +905,25 @@ func addMasterEC2Policies(p *Policy, resource stringorslice.StringOrSlice, clust
 				},
 			},
 		},
+		&Statement{
+			Effect: StatementEffectAllow,
+			Action: stringorslice.Of(
+				"ec2:AttachVolume",                  // aws.go
+				"ec2:AuthorizeSecurityGroupIngress", // aws.go
+				"ec2:CreateRoute",                   // aws.go
+				"ec2:DeleteRoute",                   // aws.go
```

Comment:
These aren't related to volumes and the EBS CSI driver so I'm curious why they're being included here.

Reply:
Looks like a mistake to me. But I am planning on moving the EBS specific stuff into a dedicated function to support IRSA. Can remove it as part of that PR.

Reply:
yes I just copied the existing KubernetesCluster policy without questioning it